packages/perseus-core/src/parse-perseus-json/perseus-parsers/numeric-input-widget.ts

```diff
@@ -40,7 +40,7 @@ export const parseNumericInputWidget: Parser<NumericInputWidget> = parseWidget(
                 // the data, simplify this.
                 value: optional(nullable(number)),
                 status: string,
-                answerForms: optional(array(parseMathFormat)),
+                answerForms: defaulted(array(parseMathFormat), () => undefined),
```

Comment:
So is the different here that originally it was possible for the `answerForms` key to not exist but now it must exist but it's value is possibly `undefined`?

Reply:
The difference is that `null` was previously rejected as invalid, but now it's accepted and converted to `undefined`.

`optional(T)` allows either `T` or `undefined`, and rejects `null`.

`defaulted(T, () => U)` will accept `T`, `undefined`, or `null` as input, and convert `null` and `undefined` to `U`.